src/js/stores/MarathonStore.js

```diff
@@ -160,6 +160,12 @@ var MarathonStore = Store.createStore({
   },
 
   processMarathonGroups: function (data) {
+    data.filterProperties = {
+      name: function (item) {
+        return item.id;
+      }
+    };
+
```

Comment:
#75 is merged. can we use any of the logic from there to replace modifying the data object here?